Bottom: fe09bc2fb6bf901d2de5ff3753c5357eb56897d0
Top:    26f963995b6a23659ece93d1457aa44e36d38d02
Author: scott snyder <sss@karma>
Date:   2020-02-12 11:47:58 -0500

.gitignore


---

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b25c15b
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+*~
